src/install/local/quick-start.md

```diff
@@ -25,32 +35,46 @@ Our automated installation process is setting up all the required dependencies l
 - Go
 - Helm
 
->_Note: Once the Engine installed your k8s context will be switched automatically to new cluster._   
+>_Note: Once the Engine installed your k8s context will be switched automatically to new cluster._
 
 Here are installation steps:
 
 1. Pull down the latest from the [MDAI infrastructure installation repo](https://github.com/DecisiveAI/mdai-inkops)
 2. Install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) for local cluster management using docker containers
 3. Run automated installation script
 ```bash
-make -f ./make/Makefile-local-recipies create-mdai
+make -f ./make/Makefile-local-recipes create-mdai
 ```
+
+<div class="warning">
+   <b>Other steps:</b>
+   While we install docker, you also need to ensure that the docker process is running. You can simply open the Docker app or start the Docker process to ensure the local runs flawlessly.
```

Comment:
The above command `make -f ./make/Makefile-local-recipes create-mdai`installs and uses `docker`, so I think the wording should change here to saying something like, "if it fails ensure the docker daemon is running".

I wasn't sure of the wordinig so just leaving this as a comment for now.

Also, saying "open the Docker app or start the Docker process" sounds MacOS specific. Could just link to some docker.io docs or include Linux by adding `systemctl start docker` (`systemctl enable docker` to start on boot) -- not universal across all Linux distros but most these days are systemd based.